### PR TITLE
post(datavolumes): Add CDI DataVolumes post

### PR DESCRIPTION
**Pull request by tripledes**
* Complete blog post.
* Set date for October 11th.

### CODE REVIEW

File: _posts/2018-10-10-CDI-DataVolumes.markdown [hunk 1]
```diff
@@ -0,0 +1,274 @@
...
+
+Alternatively, the following one-liner can also be used to achieve the same result:
+
+```shell
+$ oc create configmap kubevirt-config --from-literal feature-gates=DataVolumes -n kube-system
+```
+
+If the ConfigMap was already present on the system, just use `oc edit` to add the DataVolumes feature gate under the _data_ field like the YAML above.
+
+If everything went as expected, we should see the following log lines on the _virt-controller_ pods:
```
[the remark this answers is not in the document]
Comment (tripledes) [reply _posts/2018-10-10-CDI-DataVolumes.markdown]: @davidvossel when I set everything up, I installed KubeVirt and once everything was running I created the configmap, scale down and up the virt controller and the feature was enabled, is that what you mean by it not being *dynamic* ? I can reword it so it's more clear.

[the remark this answers is not in the document]
Comment (davidvossel) [reply _posts/2018-10-10-CDI-DataVolumes.markdown]: I see. I'm surprised just cycling virt-controller works though. virt-api reads these feature gates at initialization and our validation webhook is supposed to reject VM's with DataVolumes if the feature gate isn't enabled.

So, if you cycle virt-controller and virt-api after making the config change, that should fully enable DataVolume support.

[the remark this answers is not in the document]
Comment (tripledes) [reply _posts/2018-10-10-CDI-DataVolumes.markdown]: Thanks! Will fix that and update the PR.

**Comment by tripledes** [comment]
@davidvossel just added a NOTE mentioning the need to cycle virt-{api,controller}.

**Comment by tripledes** [comment]
[(2) DataVolumes blog post [epic-supportability]](https://trello.com/c/maW8tfR4/88-2-datavolumes-blog-post-epic-supportability)

**Comment by scollier** [comment]
@tripledes LGTM. Thanks for getting on those changes so quickly.

**Pull request merged**